CRM/Contribute/Page/Tab.php

```diff
@@ -72,16 +72,13 @@ public static function recurLinks(int $recurID, $context = 'contribution') {
       ];
     }
 
-    $links[CRM_Core_Action::DISABLE] = [
-      'name' => ts('Cancel'),
-      'title' => ts('Cancel'),
-      'ref' => 'crm-enable-disable',
-    ];
-
     if ($paymentProcessorObj->supports('cancelRecurring')) {
-      unset($links[CRM_Core_Action::DISABLE]['extra'], $links[CRM_Core_Action::DISABLE]['ref']);
-      $links[CRM_Core_Action::DISABLE]['url'] = "civicrm/contribute/unsubscribe";
-      $links[CRM_Core_Action::DISABLE]['qs'] = "reset=1&crid=%%crid%%&cid=%%cid%%&context={$context}";
+      $links[CRM_Core_Action::DISABLE] = [
+        'name' => ts('Cancel'),
+        'title' => ts('Cancel'),
+        'url' => 'civicrm/contribute/unsubscribe',
+        'qs' => "reset=1&crid=%%crid%%&cid=%%cid%%&context={$context}",
+      ];
```

Comment:
Does the above change mean we load `Cancel` link only for processors supporting `cancelRecurring`?

But from the description -

>After
>Always display the cancellation form. Eg. for a recur with payment processor:

it seems the cancel link should always be visible and condition should be checked on the form, i.e, for `Send cancellation request to processor` field? Is that correct?

With this PR applied, I see the Cancel link is not displayed for processors not having `supportsCancelRecurring()`.

![image](https://user-images.githubusercontent.com/5929648/135465335-4e5f0d27-d279-45c4-8bdd-ad8c6fed3c4f.png)

@mattwire can you pls clarify?

Reply:
@jitendrapurohit You are right. I've have pushed a fix.